### PR TITLE
Add blog article about MOOCs status

### PR DESCRIPTION
**Pull request by julienrf**

### CODE REVIEW

File: blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+[Coursera](https://www.coursera.org) platforms. The benefits are twofold: we can test and improve our next courses
+internally at EPFL on [an Open edX based platform](https://courseware.epfl.ch/) before making them public, and we
+believe that by having more freedom on where we eventually publish our courses ([edX](https://www.edx.org/) or
+[Coursera](https://www.coursera.org)) you will be more likely to get the best online learning experience.
+
+Our courses content is available for free. In addition, a paid version allows you to receive a grade for the whole
+course as well as a certificate of completion. Statistics revealed that this paid version is mostly used by companies
+to train their employees. The MOOCs revenues allow us to pay one engineer (myself) and a team of teaching assistants.
+
+## What we plan to do
```
Comment (julienrf) [inline blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md]: I’m still not sure if I should use “I” or “we”, in this section.

File: blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+We believe that we were able to manage this load of learners because our assignments grading system was an automated
+process: when a learner submitted his/her work, our grading infrastructure analyzed the submission’s source code, ran
+the tests and produced detailed feedback for each observed error. In addition to this automated infrastructure, our
+team of (human!) teaching assistants was present on the discussion forums to answer other requests.
+
+Encouraged by this success, one year later we launched our second course: Principles of Reactive Programming.
+
+## Where we are
+
+We currently have 6 running courses, covering the basics of the language to more specific programming techniques:
```
Comment (SethTisue) [inline blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md]: s/covering the/ranging from the/

File: blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  Malawski, Julien Richard-Foy, Martin Odersky): teaches programming techniques for implementing scalable, resilient,
+  and responsive systems.
+
+In practice, we are able to deploy our grading infrastructure on both the [Open edX](https://open.edx.org/) and the
+[Coursera](https://www.coursera.org) platforms. The benefits are twofold: we can test and improve our next courses
+internally at EPFL on [an Open edX based platform](https://courseware.epfl.ch/) before making them public, and we
+believe that by having more freedom on where we eventually publish our courses ([edX](https://www.edx.org/) or
+[Coursera](https://www.coursera.org)) you will be more likely to get the best online learning experience.
+
+Our courses content is available for free. In addition, a paid version allows you to receive a grade for the whole
```
Comment (SethTisue) [inline blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md]: s/Our courses content/The content of our courses/

File: blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+We will keep making our material accessible for free and we will continue selling certificates of completion to
+sustain our activity.
+
+More concretely, our next steps in the roadmap are the following:
+- teach more advanced concepts (implicits, path-dependent types, etc.),
+- give ready to use recipes for common programming tasks (manipulate files, perform HTTP requests, etc.),
+- update the material to recent Scala versions and Dotty.
+
+And you, what would be your expectations for our next MOOCs?
```
Comment (SethTisue) [inline blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md]: where are readers expected to send this feedback?

Comment (julienrf) [reply blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md]: I will create a post on users.scala-lang.org and add a link to it here.

File: blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy
+title: "What’s up with the Scala MOOCs?"
```
Comment (sjrd) [inline blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md]: Hum, reading that title makes me think that you're trying to address voiced concerns about the MOOCs. Have you considered something like "News from the Scala MOOCs" or "What's new about the Scala MOOCs?".

File: blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md (hunk 1 above)
Comment (sjrd) [reply blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md]: Whatever you pick, it needs to fit the first line "and introduce my plans". I would suggest using "we", as usual. ;)

File: blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+- [Scala Specialization Capstone Project](https://www.coursera.org/learn/scala-capstone/) (Julien Richard-Foy): the
+  four previous courses are grouped in a [specialization](https://www.coursera.org/specializations/scala), and this
+  course invites students to apply the previously learned skills by implementing a large project,
+- [Programming Reactive Systems](https://www.edx.org/course/programming-reactive-systems) (Roland Kuhn, Konrad
+  Malawski, Julien Richard-Foy, Martin Odersky): teaches programming techniques for implementing scalable, resilient,
+  and responsive systems.
+
+In practice, we are able to deploy our grading infrastructure on both the [Open edX](https://open.edx.org/) and the
+[Coursera](https://www.coursera.org) platforms. The benefits are twofold: we can test and improve our next courses
+internally at EPFL on [an Open edX based platform](https://courseware.epfl.ch/) before making them public, and we
```
Comment (sjrd) [inline blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md]: edX-based with a dash, I believe.

File: blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+Our goal is to **ease the adoption of Scala** and, beyond the language, to turn new, **innovative**, programming
+techniques into **familiar** practices.
+
+We will keep making our material accessible for free and we will continue selling certificates of completion to
+sustain our activity.
+
+More concretely, the next steps in our roadmap are the following:
+- teach more advanced concepts (implicits, path-dependent types, etc.),
+- give ready to use recipes for common programming tasks (manipulate files, perform HTTP requests, etc.),
```
Comment (sjrd) [inline blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md]: ready-to-use ?

File: blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md (hunk 5 above)
Comment (julienrf) [reply blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md]: I’ve changed for “News from the Scala MOOCs”

File: blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md (hunk 1 above)
Comment (julienrf) [reply blog/_posts/2019-05-21-what-s-up-with-scala-moocs.md]: I kept “we”

**Pull request merged**